recipes/isoseq3/0.7.2/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name = "isoseq3" %}
+{% set version = "0.7.2" %}
+{% set sha256 = "bd3e272dddb6153bd949e70cb57718c92c52608bc76ced8e95a08c5ee94b52b0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: isoseq3
+  url: https://github.com/PacificBiosciences/IsoSeq3/releases/download/v{{ version }}/isoseq3 # [linux]
```

Comment:
@pb-dseifert is it possible to compile isoseq3 from source?

Reply:
IsoSeq is one of those applications that PacBio does not want to open-source at this stage.